### PR TITLE
Fix `Cargo.toml`

### LINKED ISSUE #744: Change authors in config file
## Description

Change the authors to `"IOTA Stiftung"` in the config file. 

## Motivation

Should be consistent across projects. Bee & stronghold use `"IOTA Stiftung"` as authors. 

## Open questions (optional)

N/A

## Are you planning to do it yourself in a pull request?

Maybe.

### PR DESCRIPTION
**Pull request by jyhi**
This PR tries to improve the quality of `Cargo.toml`s.

Fix #744.

### CODE REVIEW

File: bindings/nodejs/native/Cargo.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-[workspace]
-
 [package]
-name = "iota-client"
+name = "iota-client-nodejs"
```
Comment (Thoralf-M) [inline bindings/nodejs/native/Cargo.toml]: Here it's the same issue as for wallet.rs `The crate name will be read from the npm_package_name environment variable. If the package name includes a namespace (@namespace/package), the namespace will be removed when matching the crate name (package).`

File: bindings/wasm/native/Cargo.toml [hunk 2]
```diff
@@ -1,16 +1,12 @@
-[workspace]
-
 [package]
-name = "client-wasm"
+name = "iota-client-wasm"
```
Comment (Thoralf-M) [inline bindings/wasm/native/Cargo.toml]: I think here it's also related to the npm package name, but needs to be verified

File: bindings/nodejs/native/Cargo.toml (hunk 1 above)
Comment (jyhi) [reply bindings/nodejs/native/Cargo.toml]: So it's `cargo-cp-artifact` having such behavior. iota.rs does not use it during the Node.js / WASM bindings build. Will it be added in the future?

Comment (jyhi) [reply bindings/nodejs/native/Cargo.toml]: I'm fixing this anyways, we don't publish the crates.

File: Cargo.toml [hunk 3]
```diff
@@ -1,21 +1,12 @@
...
-  "Nathan Jaremko <nathan@jaremko.ca>",
-  "Yu-Wei Wu <wusyong9104@gmail.com>"
-]
+authors = ["IOTA Stiftung"]
 edition = "2018"
-description = "Client to use IOTA APIs"
-readme = "README.md"
+description = "The official, general-purpose IOTA client library in Rust for interaction with the IOTA network (Tangle)"
 repository = "https://github.com/iotaledger/iota.rs"
-homepage = "https://www.iota.org"
```
Comment (Thoralf-M) [inline Cargo.toml:16]: Why do you want to remove this?

Comment (jyhi) [reply Cargo.toml:16]: I think it's some copy-pasta mistakes -- wallet doesn't have one either.

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**